tokio/src/sync/notify.rs

```diff
@@ -558,6 +605,21 @@ impl Notify {
     // Alias for old name in 0.x
     #[cfg_attr(docsrs, doc(alias = "notify"))]
     pub fn notify_one(&self) {
+        self.notify_with_strategy(NotifyOneStrategy::Fifo);
+    }
+
+    /// Notifies the last waiting task.
+    ///
+    /// This function behaves identically to `notify_one` but using a
+    /// LIFO algorithm to notify waiters from the queue, if there are any.
```

Comment:
```suggestion
    /// This function behaves similar to `notify_one`. The only difference is that it wakes
    /// the most recently added waiter instead of the oldest waiter.
```